### ISSUE TITLE
Add MVP of culture section to project time

### ISSUE THREAD

**Issue by maxwofford** [hackedu/hack-camp#383]
This will likely include examples of scenarios we have run into previously. This should include right and wrong ways to approach different situations.

One example of this could be holding up energizers on a single hacker who is not ready to go to energizers.

"I'm confused about what I'm supposed to do , I don't know where the line is, what I'm supposed to say. @gemmabusoni." Getting people to do stuff, where is the line.

hackEDU is going to do it's own thing, feel free to do the thing yourself.

**Comment by jonleung** [comment]
Waiting on @MaxWofford to review and merging #419

**Comment by maxwofford** [comment]
#419 is blocking on merge conflicts

**Comment by jonleung** [comment]
I made a [new pull request](https://github.com/hackedu/hack-camp/pull/524) and closed the [old one](https://github.com/hackedu/hack-camp/pull/419).

I assigned the new pull request to @zachlatta.

**Comment by zachlatta** [comment]
Closed by #524.
